src/tsolvers/lasolver/LASolver.cc

```diff
@@ -855,11 +855,105 @@ bool LASolver::shouldTryCutFromProof() const {
     return ++counter % 10 == 0;
 }
 
+namespace {
+
+struct DefiningConstraint {
+    PTRef lhs;
+    opensmt::Real rhs;
+};
+
+/*
+ * Translates the set of defining constraints (linear equalities) into a suitable inner representation:
+ * - Matrix A of coefficients
+ * - a vector with right-hand-side values (as Rationals)
```

Comment:
Maybe replace these two points with a combined one mentioning the `SparseLinearSystem`?

Reply:
I'll try something..